nest-tax-backend/.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
Should there be a `.github` directory in this subdirectory?

Reply:
thanks, no it shouldn`t, removed